pkg/runtime/cli.go

```diff
@@ -92,14 +92,15 @@ func FromFlags() (*DaprRuntime, error) {
 	metricsExporter.Options().AttachCmdFlags(flag.StringVar, flag.BoolVar)
 
 	// Finally parse the CLI flags!
-	flag.Parse()
+	// Ignore errors; CommandLine is set for ExitOnError.
+	flag.CommandLine.Parse(args)
 
 	// flag.Parse() will always set a value to "enableAPILogging", and it will be false whether it's explicitly set to false or unset
 	// For this flag, we need the third state (unset) so we need to do a bit more work here to check if it's unset, then mark "enableAPILogging" as nil
 	// It's not the prettiest approach, but…
 	if !*enableAPILogging {
 		enableAPILogging = nil
-		for _, v := range os.Args {
+		for _, v := range args {
```

Comment:
Here use all os.Args before, now use Args[1:], what cause the difference?

Reply:
Args[0] is the name of the command that it is running, which is `dapr` here. The aim of this for loop is to find `--enable-api-logging` flag is presented or not, so it is safe to replace with args here.